aces/Loris
# [htaccess]  Modifies rule to allow spaces in filename when downloading files.

## Brief summary of changes
- The flags B and BNP were added to the rewrite rule that redirect to `index.php`. 
- This is need as files who have spaces in the name are not longer downloading (e.g. the Document Repository)
- This issue seems to appear after the upgrade of the Apache server. More details can be found in the original issue sent to LORIS:
https://github.com/aces/Loris/issues/8706

#### Testing instructions (if applicable)

1. Go to one of the modules that manage files (e.g Document Repository)
2. Try to download a file where the filenames contains spaces.
3. The file should properly download.
4. Be sure nothing else breaks elsewhere ( this change is done in the `.htaccess` file, very low level) 

#### Link(s) to related issue(s)

* Resolves #  https://github.com/aces/Loris/issues/8706
